Classes/Replicator/NodeReplicator.php

```diff
@@ -91,4 +91,23 @@ protected function logReplicationAction(NodeInterface $nodeVariant, $message)
 
         $this->logger->log(sprintf('[NodeIdentifier: %s, TargetDimension: %s] %s', $nodeVariant->getIdentifier(), $dimensionString, $message));
     }
+
+    /**
+     * @param NodeInterface $node
+     */
+    public function similarizeProperties(NodeInterface $node)
```

Comment:
The method name should make clear, that only properties get overwritten which are considered empty.

Reply:
changed to `similarizePropertiesEmptyInOtherDimensions` - a bit more verbose but tells everything that happens